data.sql

```diff
@@ -6,3 +6,13 @@ VALUES (1, 'Agumon', '2020-2-3', 0, true, 10.23),
   (3, 'Pikachu', '2021-1-7', 1, false, 15.04),
   (4, 'Devimon', '2017-5-12', 5, true, 11);
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (5, 'Charmander', '02-08-2020', 0, false, 11),
+(6, 'Plantmon', '11-15-2021', 2, true, 5.7),
+(7, 'Squirtle', '04-02-1993', 3, false, 12.13),
+(8, 'Angemon', '06-12-2005', 1, true, 45);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (9, 'Boarmon', '06-07-2005', 7, true, 20.4),
+(10, 'Blossom', '10-13-1998', 3, true, 17),
+(11, 'Ditto', '05-04-2022', 4, true, 22);
```

Comment:
- Please, kindly ensure you've used the correct date `2022-05-14`.
- Also, kindly add a `dot` and zero to the number to meet the project requirement `22.0`